lseed.go

```diff
@@ -2,24 +2,25 @@ package main
 
 import (
 	"flag"
+	"os"
 	"os/user"
 	"path/filepath"
 	"strings"
 	"time"
 
-	"github.com/cdecker/lseed/lightningrpc"
+	"./seed"
```

Comment:
I'm really not sure about this (first time GOing), I had to use this otherwise it would always pull your Github repo's `seed/` instead of the one I modified..